core/templates/dev/head/domain/exploration/EditableExplorationBackendApiService.js

```diff
@@ -113,8 +93,36 @@ oppia.factory('EditableExplorationBackendApiService', [
       });
     };
 
+    var _getExplorationUrl = function(explorationId, applyDraft, role) {
+      if (role) {
+        return UrlInterpolationService.interpolateUrl(
+        EDITABLE_EXPLORATION_DATA_ROLE_URL_TEMPLATE, {
```

Comment:
I would expect that as well, but the new linter gave me an error hence this strange format. Let me see if I can correct.